$OpenBSD: patch-cmake_FindReadline_cmake,v 1.1 2013/05/14 09:16:25 zhuk Exp $
Zap the odd hint path suffix, making it find libreadline from base.
--- cmake/FindReadline.cmake.orig	Thu May  9 21:52:47 2013
+++ cmake/FindReadline.cmake	Thu May  9 21:56:49 2013
@@ -3,7 +3,7 @@ if(READLINE_INCLUDE_DIR AND READLINE_LIBRARY)
 	set(READLINE_FOUND TRUE)
 else(READLINE_INCLUDE_DIR AND READLINE_LIBRARY)
 	FIND_PATH(READLINE_INCLUDE_DIR readline/readline.h
-		/usr/include/readline
+		HINTS /usr/include
 	)
 	
 	FIND_LIBRARY(READLINE_LIBRARY NAMES readline)
